roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
I don't think it will hurt anything having ``/var/usrlocal/bin`` in the path. Normally I'd go ahead and remove it since it should be the same as ``/usr/local/bin`` but I'd rather not re-ci for it.